azure/logs/doc/manual_setup.md

```diff
@@ -48,4 +48,5 @@ Azure function apps have built-in triggers for Event Hub. When the triggers are
 - Add SWI_OTEL_ENDPOINT environmental variable containing the URI of the telemetry endpoint: https://api.dc-01.cloud.solarwinds.com/v1/logs
 - Save the function.
 
-
+## Logs forwarding
+Logs must be forwarded for each resource from which logs will be monitored. It can be done following [guide](logs_forwarding.md)
```

Comment:
I don't think, this is true. For example I did not need to set those activity logs forwarding for each resource, I set it once and it worked for VMs, AKS,...so I would rather write: Forward logs you want to see in <nighthawk official name> website to created event hub.